ui/dashboards/src/components/Variables/variable-model.ts

```diff
@@ -101,3 +102,42 @@ export const VARIABLE_TYPES = [
   { label: 'List', kind: 'ListVariable' },
   { label: 'Text', kind: 'TextVariable' },
 ] as const;
+
+/*
+ * Check whether saved variable definitions are out of date with current default list values in Zustand store
+ */
+export function updateVariableDefaultValues(
```

Comment:
So i think you can actually move this logic to the TemplateVariableStore so you don't need to pass in savedVariables and can also directly call `setDefinitions` there, which will automatically update the dashboard json

Reply:
happy to jump on a call to discuss if this isn't clear :)